src/yum-plugins/enabled_repos_upload.py

```diff
@@ -98,7 +101,7 @@ def report_enabled(self, consumer_id, report):
         method = '/systems/%s/enabled_repos' % self.sanitize(consumer_id)
         try:
             self.conn.request_put(method, report)
-        except (RemoteServerException, GoneException) as e:
+        except (RemoteServerException, GoneException), e:
```

Comment:
This won't work in python 3, but that's okay for now.  If we have shared code between yum and dnf when we get around to making dnf plugins we'll worry about it then.